Description: Define IS_EINVAL_EXTRA for Hurd systems
 clisp (1:2.49-10) unstable; urgency=medium
 .
   * Makes clisp compile under debuild
   * Solves https://lists.debian.org/debian-hurd/2012/06/msg00053.html)
Author: Flavio Cruz <flaviocruz@gmail.com>

Last-Update: 2015-09-23

--- clisp-2.49.orig/src/stream.d
+++ clisp-2.49/src/stream.d
@@ -3483,6 +3483,8 @@ local void clear_tty_input (Handle handl
   #define IS_EINVAL_EXTRA  ((errno==EBADF)||(errno==EACCES)||(errno==EBADRQC))
 #elif defined(UNIX_DARWIN) || defined(UNIX_FREEBSD) || defined(UNIX_NETBSD) || defined(UNIX_OPENBSD)
   #define IS_EINVAL_EXTRA  ((errno==EOPNOTSUPP)||(errno==ENOTSUP)||(errno==ENODEV))
+#elif defined(UNIX_HURD)
+  #define IS_EINVAL_EXTRA  ((errno==EOPNOTSUPP)||(errno==EMIG_BAD_ID))
 #else
   #define IS_EINVAL_EXTRA  0
 #endif
